modules/ROOT/pages/mule-error-concept.adoc

```diff
@@ -56,58 +56,60 @@ Below you can see what the core runtime’s hierarchy looks like:
 
 image::error-hierarchy.png[Error Hierarchy]
 
-All errors are either general or `CRITICAL`, the latter being so severe that they cannot be handled and will only be logged. At the top of the general hierarchy is `ANY`, which allows for matching all types under it. It’s important to note the `UNKNOWN` type, which is used when no clear reason for the failure is found. This error can only be handled through the `ANY` type to allow for specification of unclear errors in the future, without changing the existing behavior of the app.
+All errors are either general or `CRITICAL`, the latter being so severe that they cannot be handled and will only be logged. At the top of the general hierarchy is `ANY`, which allows for matching all types under it. It’s important to note the `UNKNOWN` type, which is used when no clear reason for the failure is found. This error can only be handled through the `ANY` type to allow for the specification of unclear errors in the future, without changing the existing behavior of the app.
```

Comment:
The graphic above differentiates between 'ANY' and 'CRITICAL'. However, the description below presents a *general* category:
> All errors are either general or `CRITICAL` (...) 
> At the top of the general hierarchy is `ANY`, which allows for matching all types under it. 
If we are going to categorize error types within categories, we should define a category for the `CRITICAL` type as well. It seems messy to present a  "general" category as opposed to a "critical" type. We should present categories in the graphic, or rewrite this sentence to avoid the use of "general".